helm/fcp-data-access-layer-api/values.yaml

```diff
@@ -17,7 +17,7 @@ container:
   memCpuTier: CUSTOM
   requestMemory: 50Mi
```

Comment:
bump requestMemory to 100Mi as per Dan Rs suggestion please

Reply:
done in commit 5e68f0